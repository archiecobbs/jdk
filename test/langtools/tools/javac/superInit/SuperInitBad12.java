/*
 * @test /nodynamiccopyright/
 * @bug 8194743
 * @summary Verify non-canonical record constructors only invoke this()
 *
 * @compile/fail/ref=SuperInitBad12.out -XDrawDiagnostics SuperInitBad12.java
 */
class SuperInitBad12 {
    record Record(int value) {
        Record(float x) {
        }
    }
}
