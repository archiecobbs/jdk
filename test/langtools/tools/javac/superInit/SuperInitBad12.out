SuperInitBad12.java:10:9: compiler.err.non.canonical.constructor.invoke.other.constructors: SuperInitBad12.Record
1 error
